test/unit/auth/auth.spec.ts

```diff
@@ -3214,41 +3214,6 @@ AUTH_CONFIGS.forEach((testConfig) => {
         // Make sure this doesn't throw
         jwt.verify(token, '', { algorithms: ['none'] });
       });
```

Comment:
Can we run a unit test where we force the emulator mode by setting the env variable to a fake value, and then see if we get an error when the SDK unsuccessfully attempts to connect to it?

Reply:
Added.

Reply:
Okay, in spirit of a unit test, I did it with a mock since I cannot find an address that fails reliably on all platforms (even the unrouteable `0.0.0.0` is treated as loopback on Linux).